core/src/scene/stops.cpp

```diff
@@ -0,0 +1,84 @@
+#include "stops.h"
+#include "csscolorparser.hpp"
+#include "yaml-cpp/yaml.h"
+#include <cassert>
+#include <algorithm>
+
+#define RED(col) (col % 256)
```

Comment:
A utility file with higher-level functions on colors would be useful in many places, distance conversions too; just not macros in headers :P